### ISSUE TITLE
Which `http-proxy-agent` package do you use?

### ISSUE THREAD

**Issue by XiaoxiaYao** [vitalets/google-translate-api#123]
Привет, I could not find `createHttpProxyAgent` in the [package](https://www.npmjs.com/package/http-proxy-agent). Which lib are you using?

I am using `HttpProxyAgent` for now. But the generated agent seems not working with your lib. Your lib is using an old agent somehow.
<img width="802" alt="image" src="https://github.com/vitalets/google-translate-api/assets/22899096/6201a88d-bc91-4b63-8ba4-7e0869f2c139">

**Comment by vitalets** [comment]
Hi! I was using `http-proxy-agent` **5.x** where `createHttpProxyAgent` was defined. Now updated examples and readme for using **7.x** with `HttpProxyAgent` class.
Thank you!

**Comment by XiaoxiaYao** [comment]
Is the proxy working for you? I use the same code. But it seems like the lib is not using the proxy ip, it's using my local ip.

**Comment by XiaoxiaYao** [comment]
Yep. I tried. Still not working. No one faces this issue besides me? 😓
<img width="1208" alt="image" src="https://github.com/vitalets/google-translate-api/assets/22899096/bb521924-4ca0-4524-a0bf-aa94452f2689">

**Comment by vitalets** [comment]
Interesting. Could you try this snippet to check HttpProxyAgent in isolation?:
```ts
import { HttpProxyAgent } from 'http-proxy-agent';
import fetch from 'node-fetch';

main();

async function main() {
  console.log('without proxy');
  const res1 = await fetch('https://www.httpbin.org/ip');
  console.log(await res1.text());

  console.log('with proxy');
  const agent = new HttpProxyAgent(`http://133.18.234.13:80`); // maybe proxy needs to be replaced with fresher one
  const res2 = await fetch('https://www.httpbin.org/ip', { agent });
  console.log(await res2.text());
}
```
My output is:
```
without proxy
{
  "origin": "xx.xxx.83.147"
}

with proxy
{
  "origin": "133.18.234.13"
}
```

**Comment by innocces** [comment]
also meet this problem.
